HeroTracker/script.json

```diff
@@ -1,8 +1,8 @@
 {
     "name": "HeroTracker",
     "script": "herotracker.js",
-    "version": "1.1",
-    "previousversions": [ "1.0" ],
+    "version": "1.2",
+    "previousversions": [ "1.1" ],
```

Comment:
You'd have to add "1.0" to the list of perviousversions like this: ["1.0", "1.1"] if you want people to still have access to that version.

Reply:
Thank you, I did not realize that was the syntax, I will put a change in to reflect that.